portfolio/src/main/webapp/script.js

```diff
@@ -44,15 +44,23 @@ function loadHTML(src, selector, position) {
       document.querySelector(selector).insertAdjacentHTML(position, text);
     });
 }
+
+// "Enum" for key positions used in insertAdjacentHTML() and loadHTML()
+const Adjacent = {
+  beforebegin: "beforebegin",
+  afterbegin: "afterbegin",
+  beforeend: "beforeend",
+  afterend: "afterend"
```

Comment:
You don't have to change but a general style nit to consider: typically, enum items are capitalized. Like 
``` 
{ 
   BEFORE_BEGIN: "beforebegin"
}
```

Reply:
Gotcha :) Will do.